op-node/rollup/derive/batch.go

```diff
@@ -451,8 +533,13 @@ func (b *BatchData) decodeTyped(data []byte) error {
 	case BatchV1Type:
 		b.BatchType = BatchV1Type
 		return rlp.DecodeBytes(data[1:], &b.BatchV1)
-	case BatchV2Type:
-		b.BatchType = BatchV2Type
+	case BatchV2V1Type:
```

Comment:
@ImTei  We use batch type for batchV2 type. Therefore batch type and batchV2 type will be equal.